addons/web/static/tests/core/commands/command_palette_tests.js

```diff
@@ -1595,3 +1595,60 @@ QUnit.test("checks that href is correctly used", async (assert) => {
     await nextTick();
     assert.verifySteps(["command_with_link_clicked"]);
 });
+
+QUnit.test("searchValue must not change without edition", async (assert) => {
+    const provideDef = makeDeferred();
+    const debounceDef = makeDeferred();
+
+    let count = 0;
+    patchWithCleanup(browser, {
+        clearTimeout: () => {},
+        setTimeout: async (later) => {
+            count += 1;
+            if (count === 2) {
```

Comment:
suggestion: set a flag to true just before typing deb (more robust than counting, probably)